internal/execution/queries.go

```diff
@@ -0,0 +1,44 @@
+package execution
+
+import (
+	"context"
+	_ "embed"
+	"slices"
+
+	"github.com/zitadel/zitadel/internal/api/authz"
+	"github.com/zitadel/zitadel/internal/query"
+)
+
+type Queries interface {
+	ActiveInstances() []string
+	GetNotifyUserByID(ctx context.Context, shouldTriggered bool, userID string) (*query.NotifyUser, error)
+	TargetsByExecutionID(ctx context.Context, ids []string) (execution []*query.ExecutionTarget, err error)
+	GetInstanceFeatures(ctx context.Context, cascade bool) (_ *query.InstanceFeatures, err error)
+
+	InstanceByID(ctx context.Context, id string) (instance authz.Instance, err error)
+}
+
+type ExecutionsQueries struct {
+	Queries
+}
+
+func NewExecutionsQueries(
+	baseQueries Queries,
+) *ExecutionsQueries {
+	return &ExecutionsQueries{
+		Queries: baseQueries,
+	}
+}
+
+func (q *ExecutionsQueries) ActiveInstancesWithFeatureFlag(ctx context.Context) []string {
```

Comment:
either my IDE and global search are stupid or the code is not used

Reply:
You're right, seems as I had quite a confusion of the queries, cleanup the old interface